test/e2e/page-objects/pages/swap/swap-page.ts

```diff
@@ -41,6 +80,36 @@ class SwapPage {
     console.log('Advanced Settings page is loaded');
   }
 
+  async clickOnMoreQuotes(): Promise<void> {
+    await this.driver.clickElement({
+      text: `More quotes`,
+      tag: 'button',
+    });
+  }
+
+  async checkQuote(quote: SwapQuote): Promise<void> {
+    await this.driver.waitForSelector({
+      text: `${quote.totalCost} total cost`,
+      tag: 'p',
+    });
+    await this.driver.waitForSelector({
+      text: `${quote.receivedAmount} receive amount`,
+      tag: 'p',
+    });
+    await this.driver.waitForSelector({
+      text: quote.estimatedTime,
+      tag: 'p',
+    });
+    await this.driver.waitForSelector({
+      text: quote.provider,
+      tag: 'p',
+    });
+  }
+
+  async closeQuotes(): Promise<void> {
+    await this.driver.clickElement('header button');
```

Comment:
a couple of comments around this:
- could we define a selector above, and then use the defined selector, instead of hardcoding it here?
- could we use a more specific selector if possible?
- could we use the `clickElementAndWaitToDisappear` method instead? this can help us mitigate flakiness